embulk-standards/src/main/java/org/embulk/standards/LocalFileInputPlugin.java

```diff
@@ -131,10 +131,11 @@ public void cleanup(TaskSource taskSource,
 
             int maxDepth = Integer.MAX_VALUE;
             Set<FileVisitOption> opts;
-            if( task.getFollowSymlinks() ) {
+            if(task.getFollowSymlinks()) {
```

Comment:
really style nit: if we accept the fix below, add a space just after `if` in passing.